src/components/AddDependencySearchResult/AddDependencySearchResult.js

```diff
@@ -127,20 +128,54 @@ class AddDependencySearchResult extends PureComponent<Props> {
 
     const downloadNumColor = getColorForDownloadNumber(hit.downloadsLast30Days);
 
+    // We have to use a custom highlight
+    // to unescape the html
+    const CustomHighlight = connectHighlight(
+      ({ highlight, attribute, hit, highlightProperty }) => {
+        const highlights = highlight({
+          highlightProperty: '_highlightResult',
+          attribute,
+          hit,
+        });
+
+        return highlights.map((part, i) => {
+          // Run the parser on each of the parts
+          const unescaped = (new DOMParser().parseFromString(
+            `<span>${part.value}</span>`,
+            'text/html'
+          ): any).body.textContent;
+
+          // If the part is highlighted, wrap in <mark> for the highlight
+          // Otherwise just render the part in a <span>
+          return part.isHighlighted ? (
+            <mark key={i} className="ais-Highlight-highlighted">
+              {unescaped}
+            </mark>
+          ) : (
+            <span key={i}>{unescaped}</span>
+          );
+        });
+      }
+    );
+
     return (
       <Wrapper>
         <Header>
           <Title>
             <ExternalLink href={npmLink}>
-              <Name size="small">{hit.name}</Name>
+              <Name size="small">
+                <CustomHighlight attribute="name" hit={hit} />
+              </Name>
```

Comment:
I think we can skip highlighting the name. It should be invisible with the current styling (and I actually prefer not to have visible styling for the titles, feels like it would be distracting)